t3f/ops.py

```diff
@@ -793,29 +793,39 @@ def frobenius_norm_squared(tt, differentiable=False):
   """Frobenius norm squared of a TensorTrain (sum of squares of all elements).
 
   Args:
-    tt: `TensorTrain` object
+    tt: `TensorTrain` or `TensorTrainBatch` object
```

Comment:
Can we add tensortrainbatch mention into the first line of docstring and still respect the 80 symbols limit?